packages/stats/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "extends": "@tracespace/config/typescript.json",
+  "references": [
+    { "path": "../parser" },
+  ],
+  "compilerOptions": {
+    "composite": true,
+    "emitDeclarationOnly": true,
+    "rootDir": "src",
+    "outDir": "lib"
+  },
+  "include": ["src"],
+  "exclude": ["**/__tests__/**"]
```

Comment:
`"exclude": ["**/__tests__/**"]` was a mistake on my part; we should definitely not exclude tests